ui/src/components/DataTable/index.jsx

```diff
@@ -76,49 +99,83 @@ export default class DataTable extends Component {
     }
   };
 
+  handleChangePage = (event, page) => {
+    this.setState({ page });
+  };
+
   render() {
     const {
+      classes,
       items,
       columnsSize,
       renderRow,
       headers,
       sortByHeader,
       sortDirection,
       noItemsMessage,
+      isPaginate,
     } = this.props;
     const colSpan = columnsSize || (headers && headers.length) || 0;
+    const { page } = this.state;
+    const rowsPerPage = 5;
```

Comment:
Instead of setting `rowsPerPage` here, add a prop `rowsPerPage` and have it default to 5.